RELEASE/scripts/autoscend/auto_choice_adv.ash

```diff
@@ -759,6 +759,9 @@ boolean auto_run_choice(int choice, string page)
 		case 1525:
 			dartChoiceHandler(choice, options);
 			break;
+		case 1528: // Eye-Eye-Eye!
+			run_choice(1); // one button, click it
+			break;

```

Comment:
This is definitely not the way this should be handled.

If you are attempting to add support for a new path which is currently unhandled I suggest you look at how similar functionality is added for other paths in the past first & build support in a similar manner.

Reply:
I've been given a workaround and I can keep this in my copy of the code. This pull request can be abandoned if you prefer. Sorry.
At first glance would this be nearer the proper way of doing this? 
1. Create 11_things_I_hate_abiut_U.ash. 
2. Add an in_11TIHAU() function. 
3. Add a processEyeEyeEye() function that clicks the button when called. 
4. Add the imports of the new ash along with the other challenge paths. 
5. In auto_choice_adv.ash add the function call processEyeEyeEye to the switch statement for case 1528.
Autoscend copes with I8U well apart from that. I don't feel qualified to add consumption or pulls, but these changes would add the framework for others.
What do you think?